eng/tools/lint-diff/src/markdown-utils.ts

```diff
@@ -200,7 +200,7 @@ export function getDefaultTag(markdownContent: string): string {
 
   const lh = codeBlockMap[latestHeader];
   if (lh) {
-    const latestDefinition = YAML.load(lh.literal!) as undefined | { tag: string };
+    const latestDefinition = YAML.load(lh.literal!, { schema: YAML.JSON_SCHEMA }) as undefined | { tag: string };
```

Comment:
```suggestion
    const latestDefinition = YAML.load(lh.literal!, { schema: YAML.JSON_SCHEMA }) as undefined | { tag: string };
```
```suggestion
    const latestDefinition = YAML.load(lh.literal!, { schema: YAML.FAILSAFE_SCHEMA }) as undefined | { tag: string };
```

Let's go all the way and just use the FAILSAFE_SCHEMA unless we know we need more.

https://teams.microsoft.com/l/message/19:987876ab02634aaeaded0fb0de9f338a@thread.skype/1744173886127?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1744162492642&teamName=Azure%20SDK&channelName=AutoRest%20-%20Engineering&createdTime=1744173886127

Reply:
From the conversation...

> I don't think we want anything getting interpreted as a number or boolean 

Makes sense!